python/ycm/vimsupport.py

```diff
@@ -503,3 +518,27 @@ def ReplaceChunk( start, end, replacement_text, line_delta, char_delta,
 
   new_line_delta = replacement_lines_count - source_lines_count
   return ( new_line_delta, new_char_delta )
+
+
+def InsertNamespace( namespace ):
+  if VariableExists( 'g:ycm_cs_insert_namespace_function' ):
+    function = GetVariableValue( 'g:ycm_cs_insert_namespace_function' )
+    SetVariableValue( "g:ycm_namespace", namespace )
```

Comment:
Is this so that the user function can read it? If so, why not use a function param?